ast/src/main/java/com/graphicsfuzz/common/ast/TranslationUnit.java

```diff
@@ -184,6 +194,21 @@ public TranslationUnit clone() {
         .collect(Collectors.toList());
   }
 
+  public VariablesDeclaration getUniformDeclaration(String name) {
```

Comment:
Can you add a JavaDoc comment for this method, emphasising that a precondition of the method is that the uniform exists?

Also, can you fix the method so that it handles multiple and zero uniforms in the same variables declaration?

For example, in:

`uniform float a, b;`

it looks like your implementation would not currently find `b`.

I also wonder whether the method might crash on:

`uniform float;`

which is weird but legal.

Please make sure you add tests for both of these cases.

Reply:
Added the comment and the tests for both cases. The method seemed to work fine with zero uniforms in the variables declaration but didn't work with multiple uniforms. A fix to the latter case added.